### PR TITLE
fix(bug): include a missing empty line

### PR DESCRIPTION
**Pull request by TheGiraffe3**
This PR addresses the error code pull requests have been getting.

## Summary
Added an empty line.

## Testing Done
Purely a stylistic change.

### CODE REVIEW

File: source/image/Sprite.cpp [hunk 1]
```diff
@@ -20,6 +20,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "../Screen.h"
 
 #include "../opengl.h"
+
```
Comment (mOctave) [inline source/image/Sprite.cpp:23]:
```suggestion
#include "../Screen.h"
#include "../opengl.h"
```

**Review by mOctave — COMMENTED** [review]
Not 100% sure on this one, but I almost wonder if the empty line was in the wrong place, rather than missing... I'd check with a code reviewer/dev.

**Comment by TheGiraffe3** [comment]
That's what I tried initially. As you can see from the failed check on the first commit, checks didn't like it.

**Review by mOctave — APPROVED** [review]
Okay, looking at the results of the test and the current codebase, the change looks good.

**Comment by TomGoodIdea** [comment]
It's not an _error_, just a _warning_.

Maybe we should treat stylecheck warnings as errors (and make them fail the check)?

**Comment by TheGiraffe3** [comment]
How would you suggest we do that?
It looks like the Python file already uses `break` after giving a "missing empty line after changing include style" warning.
https://github.com/endless-sky/endless-sky/blob/b130a4f76c5e9ddc5a46ebb667bbe29e448c03ae/utils/check_code_style.py#L550

**Comment by warp-core** [comment]
GameWindow.cpp and ShopPanel.cpp also have the includes for "opengl.h" and SDL on consecutive, unseparated lines like Sprite.cpp does before this PR.

**Comment by warp-core** [comment]
Looking at the style checker, it looks like there is an explicit case for "opengl.h" being grouped with some `<>` files.
The reason this case generates a warning is that the path is "../opengl.h", not "opengl.h".
So, it seems like the change should be for the style checker to treat "../opengl.h" and "opengl.h" the same.

**Comment by TheGiraffe3** [comment]
I don't think we get to configure that, because we're using somebody else's checker.

**Comment by warp-core** [comment]
> I don't think we get to configure that, because we're using somebody else's checker.

Your previous comment, https://github.com/endless-sky/endless-sky/pull/10792#issuecomment-2475792488:

> How would you suggest we do that? It looks like the Python file already uses `break` after giving a "missing empty line after changing include style" warning.
> 
> https://github.com/endless-sky/endless-sky/blob/b130a4f76c5e9ddc5a46ebb667bbe29e448c03ae/utils/check_code_style.py#L550

Links to the file containing the code for the code style check.

**Comment by TheGiraffe3** [comment]
Hmm, yes, seems I was looking at the wrong check.
So upon further delving in, it doesn't look like the Python script distinguishes between "../" and the other earning it'll give. So how would that be changed?

**Comment by warp-core** [comment]
I can think of two ways to do it:
1. Just add the similar, but slightly different, path string to the list in the style checker. This is not great because it'll probably need further maintenance in the future.
2. Modify the style checker to consider file names, irrespective of relative path, in include path strings.

**Comment by TheGiraffe3** [comment]
I don't think I'm _that_ good at Python.

**Pull request closed without merging**